app/uk/gov/hmrc/securemessage/repository/ConversationRepository.scala

```diff
@@ -56,8 +57,14 @@ class ConversationRepository @Inject()(implicit connector: MongoConnector)
           Future.successful(false)
       }
 
-  def getConversations(enrolment: Enrolment)(implicit ec: ExecutionContext): Future[List[Conversation]] =
-    find(findByEnrolmentQuery(enrolment): _*)
+  def getConversations(enrolment: Enrolment)(implicit ec: ExecutionContext): Future[List[Conversation]] = {
+    import uk.gov.hmrc.securemessage.models.core.Conversation.conversationFormat
```

Comment:
can we use this top level ? so we  might need this for another function ?

Reply:
Moved.